website/mails/mails.py

```diff
@@ -372,3 +372,8 @@ def get_english_article(word):
     'send_data_share_preprint_error_desk',
     subject='Share Error'
 )
+
+REVIEWS_SUBMISSION_CONFIRMATION = lambda provider_name: Mail(
+    'reviews_submission_confirmation',
+    subject='Confirmation of your submission to {}'.format(provider_name)
```

Comment:
@laurenbarker Did `subject='Confirmation of your submission to ${provider_name}'` not work?

Reply:
Woops, disregard. Didn't see this one done in a future commit.